conf/drupal/config/core.entity_view_display.taxonomy_term.tags.default.yml

```diff
@@ -0,0 +1,20 @@
+uuid: 34840423-518e-46c1-8e2a-8ba294ad9d9d
+langcode: en
+status: true
+dependencies:
+  config:
+    - taxonomy.vocabulary.tags
```

Comment:
Git add -p apparently only works for changed files. This was another newly created file from the config-export. I'll kill it as neither file reference the hatter theme.